feature-libs/product-configurator/rulebased/components/attribute/types/radio-button/configurator-attribute-radio-button.component.spec.ts

```diff
@@ -96,4 +124,75 @@ describe('ConfigAttributeRadioButtonComponent', () => {
       initialSelectedValue
     );
   });
+
+  describe('attribute level', () => {
+    it('should not display quantity and no price', () => {
+      component.attribute.dataType =
+        Configurator.DataType.USER_SELECTION_NO_QTY;
+      fixture.detectChanges();
+
+      CommonConfiguratorTestUtilsService.expectElementNotPresent(
+        expect,
+        htmlElem,
+        '.cx-attribute-level-quantity-price'
+      );
+    });
+
+    it('should display quantity and price', () => {
+      component.attribute.quantity = 5;
+      component.attribute.attributePriceTotal = {
+        currencyIso: '$',
+        formattedValue: '500.00$',
+        value: 500,
+      };
+      component.attribute.values[0].valuePrice = {
```

Comment:
strict mode issue

Reply:
I only wanted to stress that this line causes a strict mode issue. To fix it, we need to check if the value we want to assign the valuePrice to is defined.

Reply:
Done!